packages/Webkul/Admin/src/Resources/assets/js/components/navigation/notification.vue

```diff
@@ -0,0 +1,195 @@
+                            </div>
+                        </template>
+                        <template v-if="notification.order.status == 'processing'">
+                            <div class="notif-icon processing">
+                                <span :class="ordertype.processing.icon"></span>
+                            </div>                   
+                            <div class="notif-content">
+                                <a :href="`${orderViewUrl}${notification.order_id}`">
+                                    #{{ notification.order.id }} {{ordertype.processing.message}}
+                                </a>
+                            </div>
+                            <div class="notif-content">
+                                {{ moment(notification.order.created_at).fromNow() }}
+                            </div>
+                        </template>
+                        <template v-if="notification.order.status == 'closed'">
+                            <div class="notif-icon closed">
+                                <span :class="ordertype.closed.icon"></span>
+                            </div>                
+                            <div class="notif-content">
+                                <a :href="`${orderViewUrl}${notification.order_id}`">
+                                    #{{ notification.order.id }} {{ordertype.closed.message}}
+                                </a>
+                            </div>
+                            <div class="notif-content">
+                                {{ moment(notification.order.created_at).fromNow() }}
+                            </div>
+                        </template>
+                    </li>
+                    <li class="bottom-li">
+                        <a :href="viewAll">{{ viewAllTitle }}</a>
+                        <span @click="readAll()" class="read-all">{{ readAllTitle }}</span>
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script>
+
+export default {  
+
+    props: [
+        'getNotificationUrl',
+        'viewAll',
+        'orderViewUrl',
+        'pusherKey',
+        'pusherCluster',
+        'title',
+        'viewAllTitle',
+        'getReadAllUrl',
+        'readAllTitle'   
+    ],
+
+    data(){
+
+        return {
+            notifications: [],
+             ordertype: {
+                pending : {
+                    icon: 'pending-icon',
+                    message: 'Order Pending'
+                },
+                processing : {
+                    icon: 'processing-icon',
+                    message: 'Order Processing'
+                },
+                canceled : {
+                    icon: 'canceled-icon',
+                    message: 'Order Canceled'
+                },
+                completed : {
+                    icon: 'completed-icon',
+                    message: 'Order Completed'
```

Comment:
Please use translations instead of static text

Reply:
ok sir